content/blog/posts/for-the-love-of-the-game.md

```diff
@@ -0,0 +1,109 @@
+
+**Jacob, you’ve just touched on the modularity of O3DE. What has the O3DE’s modularity meant to both of you?**
+
+**Jacob:**  Because I don't have the skills required to create something and plug it into the engine myself, I’ve appreciated the ability to go into gems in the project manager screen and just enable or disable whatever I'm looking for. The ability to search keywords for gems that may or may not exist, or scroll through to find what I’m looking for, and then enable it off the bat, has made it so easy to use. It’s lightweight: you don't have to have a ton of features enabled that you don't want, and you’re able to turn off things that you don’t need. As things get created, and officially supported gems are added, the ability to just add and remove what you're looking for makes O3DE a pleasure to use. It hasn’t been this simple in other engines I’ve used.
+
+**Phoenix:** There’s a benefit with a lot of open source software that you don't really see in any of the other engines of being able to take something made by someone else, even if it's not official, and integrate it into the software at a native level. And you just don't see that in proprietary software.
+
+It's really nice to know that if you need a feature, there may be somebody who's already made it, even if it's not part of the engine already. And if no one's made it and it's not part of the engine, you can make it and it will work as well as anything that was officially done. You can get native support for anything and that's really fantastic.
+
+**To enable some capabilities in the Engine requires the user to change a few lines of code in the code file. How do you feel about the process of installing and/or adding gems, and how could it be improved?**
+
+**Jacob:**  While neither of us have direct, hands-on experience with adding gems and capabilities specifically in O3DE that aren’t built into the Engine, we've done similar things in other engines and programs, so going into a file and changing a few lines of code isn’t a problem. We’ve also read the documentation for how to enable capabilities in O3DE that aren’t already built in, such as [PopcornFX](https://www.popcornfx.com/plugin-o3de/) and [networking](https://docs.o3de.org/docs/user-guide/networking/), and it seems pretty straightforward and simple to do. It’s also pretty typical for programs and software that are relatively new and still being developed, so for us, that's not a problem. Integrating these capabilities into the Engine would obviously save time, but it's not a huge deal.
```

Comment:
Doc link is internal, Change to internal link, not external
Change:
`[networking](https://docs.o3de.org/docs/user-guide/networking/)`
to
`[networking](/docs/user-guide/networking/)`